content/04.baselines.md

```diff
@@ -3,7 +3,7 @@
 Before diving into a fancy thousand-layer neural network, always implement at least a simple model to establish an adequate performance baseline. 
 For example, researchers can build multinomial logistic regression or random forest models using the same software framework that is being used for DL and evaluate its classification performance. 
 This approach will help researchers with assessing the complexity of the task at hand and debugging more complex DL architectures. 
-The utility of these methods is evidenced by the recent development of hybrid models which combine DL and simpler models to improve robustness, interprability and confidence estimation [@arxiv:1803.04765; @arXiv:1805.11783].
+The utility of these methods is evidenced by the recent development of hybrid models which combine DL and simpler models to improve robustness, interprability and confidence estimation [@arxiv:1803.04765; @arxiv:1805.11783].
```

Comment:
> improve robustness, interprability and confidence

Since we use Oxford commas elsewhere, this can be changed to 

"improve robustness, interprability, and confidence"

to improve consistency.

Reply:
Good call, done.